wazo_provd_client/commands/params.py

```diff
@@ -8,22 +8,30 @@ class ParamsCommand(ProvdCommand):
     resource = 'configure'
     _headers = {'Content-Type': 'application/vnd.proformatique.provd+json'}
 
+    def _build_headers(self, kwargs):
+        headers = self._headers or {}
+        # The requests session will use self.tenant_uuid by default
+        tenant_uuid = kwargs.pop('tenant_uuid', None)
+        if tenant_uuid:
+            headers['Wazo-Tenant'] = tenant_uuid
+        return headers
```

Comment:
This method is already defined in `RESTCommand` (inherited by `ProvdCommand` and by this command)
So you can use `self._get_headers()` method directly :smiley: